plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleArtifactResolvingHelper.java

```diff
@@ -82,7 +82,7 @@ public ArtifactSpec resolve(final ArtifactSpec spec) {
     }
 
     @Override
-    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive) throws Exception {
+    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive, boolean defaultExcludes) throws Exception {
```

Comment:
OK, that makes sense. What I thought I mentioned in another thread, was that the gradle implementation shouldn't rely on the m2 repo. And also in this case, using maven-related classes to solve the problem in the gradle-implementation isn't the best approach. The filtering should happen with the gradle-api.

This and my pull-request https://github.com/wildfly-swarm/wildfly-swarm/pull/147 doesn't relate to each other apart from the fact that I also tried to solve the problem with the gradle-api and not with a custom implementation.